frameworks/cassandra/docs/node-settings.md

```diff
@@ -90,6 +90,10 @@ Placement constraints allow you to customize where Apache Cassandra nodes are de
 *   **In DC/OS CLI options.json**: `placement_constraint`: string (default: `""`)
 *   **DC/OS web interface**: `PLACEMENT_CONSTRAINT`: `string`
 
+## Rack-Aware Placement
+
+Cassandra's "rack"-based fault domain support may be enabled by specifying a placement constraint which uses the `@zone` key. For example, one could spread Cassandra nodes across a minimum of three different zones/racks by specifying the constraint `@zone:GROUP_BY:3`. When a placement constraint specifying `@zone` is used, Cassandra nodes will be automatically configured with `rack`s which match the names of the zones. If no placement constraint referencing `@zone` is configured, all nodes will be configured with a default rack of `rack1`.
```

Comment:
"constraint that uses..."

"with `racks that match..."

Reply:
Done